src/common/twitter.ts

```diff
@@ -6,7 +6,9 @@ export const truncateToTweet = (text?: string): string => {
   return text.length <= 130 ? text : `${text.substring(0, 127)}...`;
 };
 
-export const truncatePostToTweet = (post: Pick<Post, 'title'>): string => {
+export const truncatePostToTweet = (
+  post: Pick<Post, 'title'> | undefined,
```

Comment:
Why not `Partial`? (more of a question not really blocking)

Reply:
Its not object syntax